src/aks/ingress.tf

```diff
@@ -0,0 +1,32 @@
+# from Microsoft docs https://docs.microsoft.com/it-it/azure/aks/ingress-internal-ip
+resource "helm_release" "ingress" {
+  name       = "nginx-ingress"
+  repository = "https://kubernetes.github.io/ingress-nginx"
+  chart      = "ingress-nginx"
+  version    = "3.31.0"
+  namespace  = kubernetes_namespace.ingress.metadata[0].name
+
+  values = [
+    "${templatefile("${path.module}/ingress/loadbalancer.yaml", { load_balancer_ip = var.ingress_load_balancer_ip })}"
```

Comment:
Well actually the naming convention suggests to name the template file ending with `.tpl`
Indeed it's not a valid yaml file.
Something like: loadbalancer.yaml.tpl

Reply:
I can update to .tpl but changing extension we lost file formatting.
Is this convention mandatory?

Reply:
done in 27d98f28c2018277d187c9736bf20b9c8b704869